### DISCUSSION TITLE
Display Legend instead of value on Bar

### DISCUSSION THREAD

**Discussion by Vucko95** [chartjs/chartjs-plugin-datalabels#389] [General]
Hi, 
Is there a way with this Plugin to display, name that is on right side on the bar itself instead of data . 
![image](https://github.com/chartjs/chartjs-plugin-datalabels/assets/29492908/3b577bdc-54b7-481c-9378-026f2569e9b8)

**Comment by simonbrunel** [comment]
https://chartjs-plugin-datalabels.netlify.app/samples/advanced/custom-labels.html

**Comment by Vucko95** [comment]
Thanks for response, but I'm looking to get the names on the right , within the bars instead of numbers .
This it what it gets me with setting achor and aling 
![image](https://github.com/chartjs/chartjs-plugin-datalabels/assets/29492908/8b419881-c7f0-42c7-b2e8-a583dcb0805e)

**Reply by simonbrunel** [reply]
The solution is in the `formatter` option

**Comment by Vucko95** [comment]
Thank you all working
